app/assets/stylesheets/components/_footer.scss

```diff
@@ -128,3 +128,10 @@
     display: block;
   }
 }
+
+.footer__copyright {
+  margin: 0;
+  color: $color-white;
+  line-height: 0.5rem;
```

Comment:
I think this value should be greater than 1 so the text renders correctly across more than 1 line.